doc/articles/features/open-id-connect.md

```diff
@@ -0,0 +1,206 @@
+
+## Demo Endpoint
+
+This code uses the _IdentityServer_ demonstration endpoint with the following parameters:
+
+| Field     | Value                                     |
+| --------- | ----------------------------------------- |
+| Authority | `https://demo.itentityserver.io`          |
+| ClientId  | `interactive.confidential`                |
+| Secret    | `secret`                                  |
+| Scopes    | `openid profile email api offline_access` |
+
+>  Note: this endpoint allows any return uris. It's acceptable for demo purposes, but production application will usually requires to register your return addresses.
+
+## Step 0 - Install Uno + Create an Application
+
+Please refer to the [Getting Started documentation](../get-started.md) to install Uno and create a new application. For the sample below, an application named `OidcDemo` has been created.
+
+For platforms supporting it, the custom protocol "oidc-auth:" will be used.
+
+## Step 1 - Add Reference to OidcClient package
+
+Add the package [`IdentityModel.OidcClient`](https://www.nuget.org/packages/IdentityModel.OidcClient) to all relevant _head_ projects of your solution.
+
+## Step 2 - Prepare for Return Uri
+
+**iOS & MacOS**
+
+Add the `oidc-auth:` custom scheme in you `Info.plist` file.
+
+``` csharp
+<key>CFBundleURLTypes</key>
+<array>
+	<dict>
+		<key>CFBundleURLName</key>
+		<string>Authentication Callback</string>
+		<key>CFBundleURLSchemes</key>
+		<array>
+			<string>oidc-auth</string>
+		</array>
+	</dict>
+</array>
+```
+
+Alternatively, you can use the editor integrated in Visual Studio to set it, in the _advanced_ tab:
+
+![Return Url in iOS](../Assets/features/authenticationbroker/return-url-editor.png)
+
+**UWP**
+
+There's nothing special for UWP. Any return url will work.  You can force it to `oidc-auth:` if you want, but you won't have any special registration to do to support it.
+
+**WebAssembly**
+
+There's nothing special for UWP. The default _return uri_ of the platform (`WebAuthenticationBroker.GetCurrentApplicationCallbackUri()`) will work with this sample and will default to `<origin>/authentication-callback`. It should be something like `http://localhost:5000/authentication-callback` when running locally using Kestrel.
+
+## Step 4 - Prepare the UI
+
+Add the following button in your application:
+
+``` xml
+<StackPanel Orientation="Horizontal" Spacing="5">
+	<Button Click="SignIn_Clicked" x:Name="btnSignin" IsEnabled="False">Sign In</Button>
+    <Button Click="SignOut_Clicked" x:Name="btnSignout" IsEnabled="False">Sign Out</Button>
+</StackPanel>
+```
```

Comment:
```suggestion
``` xml
<StackPanel Orientation="Horizontal" Spacing="5">
	<Button Click="SignIn_Clicked" x:Name="btnSignin" IsEnabled="False">Sign In</Button>
	<Button Click="SignOut_Clicked" x:Name="btnSignout" IsEnabled="False">Sign Out</Button>
</StackPanel>
```
```